test/AWS.Deploy.CLI.IntegrationTests/SaveCdkDeploymentProject/RedeploymentTests.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.IO;
+using System.Linq;
+using Amazon.CloudFormation;
+using Amazon.ECS;
+using AWS.Deploy.CLI.Common.UnitTests.IO;
+using AWS.Deploy.CLI.Extensions;
+using AWS.Deploy.CLI.IntegrationTests.Extensions;
+using AWS.Deploy.CLI.IntegrationTests.Helpers;
+using AWS.Deploy.CLI.IntegrationTests.Services;
+using AWS.Deploy.CLI.Utilities;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using Task = System.Threading.Tasks.Task;
+
+namespace AWS.Deploy.CLI.IntegrationTests.SaveCdkDeploymentProject
+{
+    public class RedeploymentTests : IDisposable
+    {
+        private readonly HttpHelper _httpHelper;
+        private readonly CloudFormationHelper _cloudFormationHelper;
+        private readonly ECSHelper _ecsHelper;
+        private readonly App _app;
+        private readonly InMemoryInteractiveService _interactiveService;
+        private bool _isDisposed;
+        private string _stackName;
+        private readonly TestAppManager _testAppManager;
+        private readonly CommandLineWrapper _commandLineWrapper;
+
+        public RedeploymentTests()
+        {
+            _httpHelper = new HttpHelper(_interactiveService);
+
+            var cloudFormationClient = new AmazonCloudFormationClient();
+            _cloudFormationHelper = new CloudFormationHelper(cloudFormationClient);
+
+            var ecsClient = new AmazonECSClient();
+            _ecsHelper = new ECSHelper(ecsClient);
+
+            var serviceCollection = new ServiceCollection();
+
+            serviceCollection.AddCustomServices();
+            serviceCollection.AddTestServices();
+
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            _app = serviceProvider.GetService<App>();
+            Assert.NotNull(_app);
+
+            _interactiveService = serviceProvider.GetService<InMemoryInteractiveService>();
+            Assert.NotNull(_interactiveService);
+
+            _testAppManager = new TestAppManager();
+
+            _commandLineWrapper = new CommandLineWrapper(new ConsoleOrchestratorLogger(new ConsoleInteractiveServiceImpl()));
+        }
+
+        [Fact]
+        public async Task AttemptWorkFlow()
+        {
+            _stackName = $"WebAppWithDockerFile{Guid.NewGuid().ToString().Split('-').Last()}";
+            var tempDirectoryPath = new TestAppManager().GetProjectPath(string.Empty);
+            await _commandLineWrapper.Run("git init", tempDirectoryPath);
```

Comment:
Is this required? Can this test be without git repo? I don't see it is a absolute necessity to make the test work.

Reply:
Git repo is not required. Removed it.